src/store/storeHelper.js

```diff
@@ -2,6 +2,7 @@
  * Nextcloud - Tasks
  *
  * @author Raimund Schlüßler
+ * @copyright 2020 Jonas Thelemann <e-mail@jonas-thelemann.de>
```

Comment:
tiny nitpick : you probably meant to add 2021 as the copyright year.

Reply:
Seems like the missing fireworks made me miss new year. Thanks for the hint! Fixed.

Reply:
the only firework I can give you is :fireworks: :sparkler:  I hope this would suffice :slightly_smiling_face: